server/routes.js

```diff
@@ -313,6 +314,12 @@ exports.isomorphicRoutes = function isomorphicRoutes(
     sMaxAge = "900",
   }
 ) {
+  //convert Latin-1(scandic) letters to plain english in url
+  app.use((req, res, next) => {
+    req.url = deburr(decodeURIComponent(req.url));
+    next();
+  });
+
```

Comment:
can we add check first if `req.url` is scandic letter then do the above logic. this will execute even for normal URLs as well.